JS/marquee.js

```diff
@@ -0,0 +1,40 @@
+class Marquee {
```

Comment:
good implementation of the marquee as a class. Note that it doesn't work for me on the UI.

Reply:
Fixed.